.codespell.exclude

```diff
@@ -30,6 +30,7 @@
 		"rasi it maka makpes"                   # fish en papillote
 		"Far Upugf Enrel Ugt"                 # Nam Myoho Renge Kyo (Buddhist religious chant)
 		"Far Uhudf Elrep Uds"                  # Nam Myoho Renge Kyo (Buddhist religious chant)
+		"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"		# (Miswritten)
```

Comment:
The comments at the end of other lines are not there to indicate why that line is being excluded, they are present in the actual content:
https://github.com/endless-sky/endless-sky/blob/8c9be87e61929653ea8a501a35490815cfbbd076/data/korath/korath%20hails.txt#L1442C27-L1442C27
The spell checker looks through the .codespell.exclude file and ignores lines in the text it is checking that exactly match lines in this file.

```suggestion
		"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"
```